File: sem-1/JAVA/Ass-1 NR/maxinarray.java
import java.util.Scanner;

public class maxinarray {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Array Length : ");
        int num = sc.nextInt();

        int[] array = new int[num];
        int max = 0;

        for (int i = 0; i < array.length; i++) {
            
            System.out.println("Enter "+(i+1)+" Number : ");
            array[i] = sc. nextInt();

            if (array[i] > max) {
                max = array[i];
            }
        }

        System.out.println("Array Element : ");
        for (int i = 0; i < array.length; i++) {
            System.out.println(array[i]);
        }
        System.out.println("Max value : "+max);
        sc.close();
    }
}

File: sem-2/DS/Lab-4/maxinarray.c
#include <stdio.h>
void main(){
	int num,i;
	printf("Enter Number Of Element  : \n");
	scanf("%d",&num);
	
	int array[num];
	printf("Enter NUmber : \n");
	for(i=0;i<num;i++){
		scanf("%d",&array[i]);
	}
	int *p = array;
	int max= *p;
	for(i=0;i<num;i++){
		if(*(p + i)>max){
			max = *(p + i);
		}
	}
	printf("Max is :  %d",max);	
}
